src/shared-components/DataGridWrapper/DataGridWrapper.spec.tsx

```
@@ -18,6 +30,262 @@ vi.mock('react-i18next', () => ({
+          type="button"
+          data-testid="trigger-pagination-change"
+          onClick={() => {
+            if (onPaginationModelChange) {
+              onPaginationModelChange({ page: 1, pageSize: 25 });
+            }
+          }}
+        >
+          Trigger Pagination
+        </button>
+
+        <div role="grid">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<table>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)